### ISSUE TITLE
insert() does not work for multiple line text

### ISSUE THREAD

**Issue by gen6033** [OnetapInc/chromy#74]
This code works.
```js
Chromy = require("chromy")
chromy = new Chromy({visible:true})
chromy.chain().goto("https://html.com/tags/textarea/").insert("textarea", "abcdef").end()
```
I added "\n" between "abc" and "def" to the above.
```js
Chromy = require("chromy")
chromy = new Chromy({visible:true})
chromy.chain().goto("https://html.com/tags/textarea/").insert("textarea", "abc\ndef").end()
```
This does not work.

**Comment by dotneet** [comment]
Hi @gen6033!
Thank you for reporting! This bug will be resolved in next version. Please wait a while.

**Comment by dotneet** [comment]
I've published v0.5.5 just now. Please check it!

**Comment by gen6033** [comment]
It works!
Thank you @dotneet．
